### PR TITLE
fix: Increase resources

### PR DESCRIPTION
**Pull request by avi-biton**
Increase resources as the controller fail to start in production

### CODE REVIEW

File: config/manager/manager.yaml [hunk 1]
```diff
@@ -65,10 +65,10 @@ spec:
...
           limits:
-            cpu: 500m
-            memory: 128Mi
+            cpu: 1000m
+            memory: 512Mi
           requests:
-            cpu: 10m
-            memory: 64Mi
+            cpu: 1000m
+            memory: 512Mi
```
Comment (yftacherzog) [inline config/manager/manager.yaml:72]: I think we should try increasing only the `limits`, or at least increase the `requests` only a little bit.

Comment (gbenhaim) [reply config/manager/manager.yaml:72]: having them equal will place the pod in the best QOS

**Comment by gbenhaim** [comment]
lgtm

File: config/manager/manager.yaml (hunk 1 above)
Comment (yftacherzog) [reply config/manager/manager.yaml:72]: That's for sure :)
Anyhow, alright. I'm ok with that.

**Review by yftacherzog — APPROVED** [review]

**Pull request merged**